src/main/default/decisionMatrixDefinition/Assigned_Score_Decision_Matrix.decisionMatrixDefinition-meta.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<DecisionMatrixDefinition xmlns="http://soap.sforce.com/2006/04/metadata">
+    <label>Assigned Score Decision Matrix</label>
+    <processType>Bre</processType>
+    <type>Standard</type>
+    <versions>
+        <fullName>Assigned_Score_Decision_Matrix_V1</fullName>
+        <columns>
+            <columnType>Input</columnType>
+            <dataType>NumberRange</dataType>
+            <displaySequence>1</displaySequence>
+            <isWildcardColumn>false</isWildcardColumn>
+            <name>RiskMatrix</name>
+            <rangeValue>0,5,10,15,20,25,30,35,40,45,50,55,60,65,70,75,80,85,90,95,570</rangeValue>
+        </columns>
+        <columns>
+            <columnType>Output</columnType>
+            <dataType>Number</dataType>
+            <displaySequence>2</displaySequence>
+            <isWildcardColumn>false</isWildcardColumn>
+            <name>AssignedScore</name>
+        </columns>
+        <decisionMatrixDefinition>Assigned_Score_Decision_Matrix</decisionMatrixDefinition>
+        <label>Assigned Score Decision Matrix V1</label>
+        <startDate>2024-03-06T11:26:06.000Z</startDate>
+        <status>Draft</status>
```

Comment:
Please clarify if this Decision Matrix must be deployed and kept as **inactive**, and if yes - then why?